src/Map/NumField.jl

```diff
@@ -322,7 +322,7 @@ function map_data_type(T::Type{<: NfRel}, L::Type{<:Any})
   MapDataFromNfRel{elem_type(L), map_data_type(base_field_type(T), L)}
 end
 
-map_data_type(K::NfRel, L::NumField) = map_data_type(typeof(K), typeof(L))
+map_data_type(K::NfRel, L::Ring) = map_data_type(typeof(K), typeof(L))
```

Comment:
This is duplicate to line 1041

Reply:
Hm yes, all the `f(x, y) = f(typeof(x), typeof(y))` could be consolidated. But I don't have time for this clean up now.